pkg_defs/sparkles/build.sh

```diff
@@ -1 +1,2 @@
-pip install --no-deps --verbose --no-binary :all: --no-index .
\ No newline at end of file
+export SKA=/dev/null
```

Comment:
For building, I think we can set it to anything and are probably better off not setting it to an actual ska.  @taldcroft do you have any thoughts on this?

Reply:
That seems fine.  Hopefully in the next release we will have a nicer plan in place, but as long as it builds and installs and tests, then whatever.